### PR TITLE
Remove light switch from lighting-app

### PR DESCRIPTION
**Pull request by mspang**
#### Problem

Endpoint 2 in lighting-app.zap is missing the descriptor cluster which
is mandatory.

#### Change overview

Since the "light switch" sample is now available, presumably we don't
need a switch endpoint on the "light" samples, so remove it rather than
add the missing cluster.

#### Testing

Regen ZAP, compile, commission the app.

### CODE REVIEW

**Comment by mspang** [comment]
@selissia Seem OK to you ?

File: examples/lighting-app/lighting-common/lighting-app.matter [hunk 1]
```diff
@@ -1144,53 +1144,6 @@ server cluster OccupancySensing = 1030 {
   readonly global attribute int16u clusterRevision = 65533;
 }
 
-client cluster OnOff = 6 {
```
Comment (bzbarsky-apple) [inline examples/lighting-app/lighting-common/lighting-app.matter:1147]: I think the idea was in fact to have an app that has both client and server lighting bits in the same app.  We've run into multiple SDK problems with that setup that had to be fixed, so this has been a useful smoketest.

Comment (mspang) [reply examples/lighting-app/lighting-common/lighting-app.matter:1147]: Interesting.. This looks pretty weird to a user though. Does all-clusters-app capture that case?

Comment (mspang) [reply examples/lighting-app/lighting-common/lighting-app.matter:1147]: FWIW I think for our purposes we would much rather have the lighting sample be only a light. This sample is supposedly an example of a smart light. I would expect it to just be a light bulb and light bulbs are not light switches.

Comment (bzbarsky-apple) [reply examples/lighting-app/lighting-common/lighting-app.matter:1147]: all-clusters-app doesn't really have client clusters.

I don't think it's necessarily a problem to remove the switch from lighting-app.  Just please check with whoever added it.

**Review by Damian-Nordic — APPROVED** [review]

File: examples/lighting-app/lighting-common/lighting-app.matter (hunk 1 above)
Comment (mspang) [reply examples/lighting-app/lighting-common/lighting-app.matter:1147]: I have reached out in the first comment on this PR.

**Comment by selissia** [comment]
> @selissia Seem OK to you ?
Yes, this is OK. My change to this was to only to add the On/Off Switch Configuration cluster -- which has since been removed from the spec.

**Comment by mspang** [comment]
@bzbarsky-apple @andy31415

**Review by bzbarsky-apple — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

**Pull request merged**